### PR TITLE
Update README.md with more specific details about az_nohttp.

### PR DESCRIPTION
**Pull request by antkmsft**

### CODE REVIEW

**Comment by antkmsft** [comment]
@ahsonkhan

File: README.md [hunk 1]
```diff
@@ -460,7 +460,7 @@ At the heart of our SDK is, what we refer to as, [Azure Core](https://github.com
 
 - **Logging**: As our SDK performs operations, it can send log messages to a customer-defined callback. Customers can enable this to assist with debugging and diagnosing issues when leveraging our SDK code. See the [Logging SDK Operations](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core#logging-sdk-operations) section of the `Azure Core` README for more information.
 
-- **Contexts**: Contexts offer an I/O cancellation mechanism. Multiple contexts can be composed together in your application's call tree. When a context is canceled, its children are also canceled. See the [Canceling an Operation](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core#canceling-an-operation) section of the `Azure Core` README for more information.
+- **Contexts**: Contexts offer an I/O cancellation mechanism. Multiple contexts can be composed together in your application�s call tree. When a context is canceled, its children are also canceled. See the [Canceling an Operation](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core#canceling-an-operation) section of the `Azure Core` README for more information.
```
Comment (ahsonkhan) [inline README.md:463]: Please revert this change.

Comment (antkmsft) [reply README.md:463]: no idea where it came from

**Comment by ahsonkhan** [comment]
@danieljurek, @weshaggard - interesting failure in link verification:

```text
Starting: Replace Relative Readme Links with Absolute References
==============================================================================
Task         : Python script
Description  : Run a Python file or inline script
Version      : 0.167.2
Author       : Microsoft Corporation
Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/python-script
==============================================================================
C:\hostedtoolcache\windows\Python\3.7.9\x64\python.exe D:\a\_temp\2843396e-376a-4a06-8775-0267201c1fb2.py
INFO:root:Root Folder: .
INFO:root:Target Folder: .
INFO:root:Repository Id: Azure/azure-sdk-for-c
INFO:root:Build SHA: 777fcc29829d5fed582cf95f7529648bdb7c609f
INFO:root:Running Relative Link Replacement on .\README.md.
ERROR:root:'utf-8' codec can't decode byte 0x92 in position 27186: invalid start byte

##[error]The process 'C:\hostedtoolcache\windows\Python\3.7.9\x64\python.exe' failed with exit code 1
Finishing: Replace Relative Readme Links with Absolute References

```

File: README.md (hunk 1 above)
Comment (antkmsft) [inline README.md:463]:
```suggestion
- **Contexts**: Contexts offer an I/O cancellation mechanism. Multiple contexts can be composed together in your application's call tree. When a context is canceled, its children are also canceled. See the [Canceling an Operation](https://github.com/Azure/azure-sdk-for-c/tree/master/sdk/docs/core#canceling-an-operation) section of the `Azure Core` README for more information.

```
